CHANGELOG.rst

```diff
@@ -2,6 +2,25 @@
 Release Notes
 =============
 
+.. _Release Notes_6.8.2:
+
+6.8.2
+=====
+
+.. _Release Notes_6.8.2_Prelude:
+
+Prelude
+-------
+
+Release on: 2018-12-19
+
+.. _Release Notes_6.8.2_Bug Fixes:
+
+Bug Fixes
+---------
+
+- Fix a null pointer exception that would happen when the agent stopped tailing a container
```

Comment:
as a customer, do I know what `tailing a container` means? :)

Reply:
We might say `stopped reading logs from a container`

Reply:
I think we should also avoid using `null pointer exception` which does not tell us if the agent crashes or not

Reply:
```Fix a bug where the agent would crash when the agent stopped reading logs from a container``` ?

Reply:
Let's replace the second `agent` by `it`. 

`stopped reading logs from a container`: I would be confused by what this means if I did not have the context. Is this happening only when a container exits ?